corehq/apps/es/es_query.py

```diff
@@ -516,10 +518,6 @@ def normalize_result(query, result):
         if query._legacy_fields and not settings.ELASTICSEARCH_MAJOR_VERSION == 7:
             return flatten_field_dict(result, fields_property='_source')
         else:
-            # ES7 scroll for some reason doesn't include _id in the source even if it's specified
-            if (settings.ELASTICSEARCH_MAJOR_VERSION == 7
-                    and getattr(query, '_source', None) and "_id" in query._source):
-                result['_source']['_id'] = result.get('_id', None)
```

Comment:
oh nice, if this is only relevant to the scroll API, I like having it there more